sdkproject/Assets/Mapbox/Unity/Map/AbstractMapVisualizer.cs

```diff
@@ -274,6 +274,28 @@ private void Factory_OnTileError(object sender, TileErrorEventArgs e)
 			}
 		}
 
+		public void ReregisterAllTiles()
```

Comment:
Rename method to RegisterAllTiles? Why does it need to be re-register?

Reply:
I felt like the name `RegisterAllTiles` implies a given list. While in this method we're reregisterin tiles (that already has been registered to map visualizer before) to factories.

Reply:
@brnkhy The method is doing nothing but registering all tiles, calling it re-register might be misleading. Re-register to me means this function will take care of all logic that is required for registering tiles that are already registered.